app/src/components/main/grid/gridBody/gridRow/gridRow.scss

```diff
@@ -110,3 +123,55 @@
     transform: scaleY(-1);
   }
 }
+
+.log {
```

Comment:
`Grid` is a common component, it shouldn't know about its' usage. Please move log-related styles to another place.

Reply:
We can't change styles for accordion's elements from `LogsGrid` component because our classNames bound. How can I bypass this?